src/textual/widgets/_button.py

```diff
@@ -1,67 +1,75 @@
 from __future__ import annotations
 
-from rich.align import Align
-from rich.console import Console, ConsoleOptions, RenderResult, RenderableType
-from rich.style import StyleType
+from typing import cast
+
+from rich.console import RenderableType
+from rich.text import Text
 
 from .. import events
 from ..message import Message
 from ..reactive import Reactive
 from ..widget import Widget
 
 
-class ButtonPressed(Message, bubble=True):
-    pass
-
-
-class Expand:
-    def __init__(self, renderable: RenderableType) -> None:
-        self.renderable = renderable
-
-    def __rich_console__(
-        self, console: Console, options: ConsoleOptions
-    ) -> RenderResult:
-        width = options.max_width
-        height = options.height or 1
-        yield from console.render(
-            self.renderable, options.update_dimensions(width, height)
-        )
-
+class Button(Widget, can_focus=True):
+    """A simple clickable button."""
+
+    CSS = """
+    
+    Button {
+        width: auto;
+        height: 3;
+        padding: 0 2;
+        background: $primary;
+        color: $text-primary;
+        content-align: center middle;
+        border: tall $primary-lighten-3;        
+        margin: 1;
```

Comment:
IMO Textual widgets should have 0 margin by default, and it's up to the user of the widget if they want to supply margin based on the context they wish to use the widget in.

Reply:
Without margin on the vertical, the buttons bump up against each other on the vertical axis, which doesn't look good. But you don't get that on the horizontal axis. So I've set margin to 1 0.

This CSS is a starting point though. Plenty of tweaking to do.